nautobot/extras/datasources/git.py

```diff
@@ -73,12 +82,45 @@ def pull_git_repository_and_refresh_data(repository_pk, request, job_result_pk):
         )
         job_result.set_status(JobResultStatusChoices.STATUS_ERRORED)
         job_result.save()
-        return
+        return job_result, None
+
+    return job_result, repository_record
```

Comment:
Please return a [`collections.namedtuple`](https://docs.python.org/3/library/collections.html#collections.namedtuple) (e.g. named `GitJobResult` or something) here instead of a naked tuple, so that the return value is self-documenting.